migrations/updateUserProfilesFromProjects/copyUserDataFromProjectsToObservations.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * name : copyUserDataFromProjectsToObservations.js
+ * author : Saish Borkar
+ * created-date : 26-nov-2024
+ * Description : Migration script for updating userProfile in observation & observationSubmission based on project information
+ */
+
+const path = require("path");
+const fs = require("fs");
+const _ = require("lodash");
+const { MongoClient } = require("mongodb");
+
+const rootPath = path.join(__dirname, "../../");
+require("dotenv").config({ path: rootPath + "/.env" });
+
+const mongoUrl = process.env.MONGODB_URL;
+const dbName = mongoUrl.split("/").pop();
+const url = mongoUrl.split(dbName)[0];
+
+(async () => {
+  const connection = await MongoClient.connect(url, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  });
+  const db = connection.db(dbName);
+
+  try {
+    console.log('Migration started...')
+    let observationIdsUpdated = [];
+    // Get all observations with referenceFrom "project"
+    const observationDocuments = await db
+      .collection("observations")
+      .find({ referenceFrom: "project" })
+      .toArray();
```

Comment:
@borkarsaish65 Project only the information which is needed, here only _id

Reply:
done